### PR TITLE
Task/ultralight parser

### PR DESCRIPTION
**Pull request by dmoranj**
Implements #1

### CODE REVIEW

File: .gjslintrc [hunk 1]
```diff
@@ -5,4 +5,5 @@
 --nomultiprocess
 --debug_indentation
 --time
+--disable=210,220
 --max_line_length=120
```
Comment (gtorodelvalle) [inline .gjslintrc]: Final blank line appreciated... :p

File: .jshintrc [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
+    "it": true,
+    "expect": true,
+    "before": true,
+    "after": true,
+    "beforeEach": true,
+    "afterEach": true,
+    "mock": true,
+    "sinon": true
+  }
+}
```
Comment (gtorodelvalle) [inline .jshintrc]: Final blank line appreciated... :p

File: Gruntfile.js [hunk 3]
```diff
@@ -150,7 +150,7 @@ module.exports = function(grunt) {
 
         dox: {
             options: {
-                title: 'iotagent-ul documentation'
+                title: 'iotagent-thinking-things documentation'
```
Comment (gtorodelvalle) [inline Gruntfile.js]: Is this reference to Thinking Things right? :)

File: package.json [hunk 4]
```diff
@@ -25,24 +25,28 @@
...
-    "proxyquire": "0.5.1",
-    "grunt-mkdir": "~0.1.1",
-    "grunt-contrib-clean": "~0.5.0",
-    "grunt-githooks": "~0.3.1"
+    "timekeeper": "0.0.5",
+    "mongodb": "*",
+    "nock": "*"
   },
   "keywords": []
 }
```
Comment (gtorodelvalle) [inline package.json]: Final blank line appreciated... :p

File: test/.jshintrc [hunk 5]
```diff
@@ -26,5 +27,9 @@
...
     "mock": true,
     "sinon": true
-  }
-}
+  },
+  "predef":
+  [
+    "describe", "beforeEach", "afterEach", "it"
+  ]
+}
```
Comment (gtorodelvalle) [inline test/.jshintrc]: Final blank line appreciated... ;)

**Comment by gtorodelvalle** [comment]
LGTM on my side apart from some minor "final blank line appreciated" comments... ;)

File: Gruntfile.js (hunk 3 above)
Comment (dmoranj) [inline Gruntfile.js]: Typo. I copied part of this file from the TT repo.

**Pull request merged**